### PR TITLE
Rename the email parameter to contact

### PR DESCRIPTION
**Pull request by adrianclay**
The variable passed in will in future be either a phone number or email address, which we generalise as contact.

### CODE REVIEW

File: spec/lib/user_spec.rb [hunk 1]
```diff
@@ -49,12 +49,12 @@
   context 'avoiding duplicate usernames' do
     before do
       srand(2)
-      User.new.generate(email: 'foo@bar.gov.uk')
+      User.new.generate(contact: 'foo@bar.gov.uk')
     end
 
     let!(:user) do
       srand(2)
```
Comment (emileswarts) [inline spec/lib/user_spec.rb]: What does this srand do?

Comment (adrianclay) [reply spec/lib/user_spec.rb]: [Seeds the system pseudo-random number generator, Random::DEFAULT, with number. The previous seed value is returned.](http://ruby-doc.org/core-2.0.0/Random.html#method-c-srand)

Comment (emileswarts) [reply spec/lib/user_spec.rb]: Feels like this User class has many responsibilities.
Would opt for having a password generator class on it's own, but can be a refactoring for the future.

File: lib/user.rb [hunk 2]
```diff
@@ -30,12 +30,12 @@ def password_from_word_list
     WORD_LIST.sample(3).map(&:capitalize).join
   end
 
-  def create_user(email)
+  def create_user(contact)
```
Comment (emileswarts) [inline lib/user.rb]: This is referred to as a model in the tests, surprising to me to see a create_user method on this object.

**Pull request merged**